src/components/App/App.js

```diff
@@ -24,37 +25,74 @@ class App extends Component {
   }
 
   addPokemon = (queriedPokemon) => {
-    this.validatePokemonData(queriedPokemon)
+    const foundPokemon = this.validatePokemonData(queriedPokemon)
+    this.setState({
+      foundPokemon: [foundPokemon]
+    })
     //set state with the validated pokemon
   }
 
   validatePokemonData = (queriedPokemon) => {
     const lowerCaseInput = queriedPokemon.queriedPokemon.toLowerCase()
-
-    const verifiedName = this.state.pokeDex.find(pokemon => {
+    
+    const verifiedName = this.state.pokeDex.find((pokemon, index) => {
       let lowerCaseName = pokemon.name.toLowerCase()
-      if (lowerCaseName.includes(lowerCaseInput) && lowerCaseInput !== '')  {
+
+      if (lowerCaseName.includes(lowerCaseInput) && lowerCaseInput !== '' && lowerCaseInput !== undefined) {
         return pokemon
+      } else if (parseInt(lowerCaseInput)){
+        if (index+1 === parseInt(lowerCaseInput)){
+          return pokemon
+        }
       }
-    })
+    });
+
+    console.log(verifiedName)
 
     if (verifiedName === undefined) {
-      console.log('No Good NAMe!')
-    } else {
+      return this.setState({ error: "Not  a valid name, try again" })
+      console.log('No Good Name!')
+    } else { 
       console.log(verifiedName, 'IT WORKSSSSSSS!!!!')
       return verifiedName
     }
+   
   }
 
   // pass the validated query through addPokemon
 
+  getPokemonImage = (id) => {
+    
+    let pokemonImage = `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/${id}.png`
+   
+    return pokemonImage
+
+  }
+
```

Comment:
This is a great! Awesome logic and work getting all of the images to render on screen

Reply:
Fun feature!